nemo/collections/nlp/models/text_classification/text_classification_model.py

```diff
@@ -166,24 +170,34 @@ def setup_test_data(self, test_data_config: Optional[DictConfig]):
         self._test_dl = self.__setup_dataloader(cfg=test_data_config)
 
```

Comment:
def setup_test_data should use setup_dataloader_from_config

Reply:
fixed.